utils/test/testinput/gdas_viirsaod2ioda.yaml

```diff
@@ -0,0 +1,11 @@
+provider: VIIRSAOD
```

Comment:
You should probably add a reference test ... We found a few bugs on our side when we did this.